jbpm-xes/pom.xml

```diff
@@ -27,13 +27,29 @@
   <name>jBPM :: XES Exporter</name>
   <description>jBPM XES Exporter</description>
   <artifactId>jbpm-xes</artifactId>
+  
+  <properties>
+    <version.commons-cli>1.4</version.commons-cli>
+    <version.com.h2database>1.4.193</version.com.h2database>
```

Comment:
Don't we want to upgrade this globally? H2 we use right now is quite old.

Reply:
upgrading h2 and commons-cli, perhaps can be done at kie-parent, instead of here, but Cristiano was concerned about the implications in other modules